### PR TITLE
Add ko Korean language

### PR DESCRIPTION
**Pull request by miumida**
Add ko Korean language

### CODE REVIEW

File: responses/ko/HassClimateGetTemperature.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+language: ko
+responses:
+  intents:
+    HassClimateGetTemperature:
+      default: "TODO: {{ state.state_with_unit }}"
```
Comment (limitless00net) [inline responses/ko/HassClimateGetTemperature.yaml:5]: `      default: "{{ state.state_with_unit }}"`

**Comment by miumida** [comment]
Maybe it's not progressing because there is no language reader? I'm waiting for it to be merged. If I'm wrong, please tell me how to fix it.

**Comment by synesthesiam** [comment]
@miumida Is the plan to add translations in a follow-up PR?

**Comment by limitless00net** [comment]
> @miumida Is the plan to add translations in a follow-up PR?

I will do that.

**Comment by limitless00net** [comment]
> @miumida Is the plan to add translations in a follow-up PR?

@synesthesiam Added initial translations #1390

**Comment by miumida** [comment]
Yes, we have a plan. There are other people in the Korean community who are interested.

**Pull request merged**